fjord/feedback/tests/test_views.py

```diff
@@ -26,6 +26,17 @@ def test_sad_redirect(self):
 class TestFeedback(TestCase):
     client_class = LocalizingClient
 
+    def test_xframe_header(self):
+        """Feedback form should *always* have X-FRAME-OPTIONS: DENY
+
+        This is because the feedback form can do various magic things
+        that shouldn't ever be framed.
+
+        """
+        url = reverse('feedback', args=(u'firefox',))
+        resp = self.client.get(url)
+        eq_(resp['X-Frame-Options'], 'DENY')
```

Comment:
I'd also check for 200 OK, since your 40x/50x page probably has different X-Frame-Options properties.